morphia/src/main/java/org/mongodb/morphia/Morphia.java

```diff
@@ -255,7 +266,7 @@ public synchronized Morphia mapPackage(final String packageName, final boolean i
      * @return this
      */
     public Morphia mapPackageFromClass(final Class clazz) {
-        return mapPackage(clazz.getPackage().getName(), false);
+        return mapPackage(clazz.getPackage().getName(), false, true);
```

Comment:
You're probably right on this method.  But, yes, in general I think the option to map subpackages should live on MapperOptions.